test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/VerifyCommandTests.cs

```diff
@@ -280,5 +282,611 @@ public async Task VerifyCommand_VerifyOnPackageSignedWithoutAllowedCertificateFa
                 verifyResult.AllOutput.Should().Contain(_noMatchingCertErrorCode);
             }
         }
+
+        [CIOnlyTheory]
+        [InlineData("true", true)]
+        [InlineData("true", false)]
+        [InlineData("false", true)]
+        [InlineData("false", false)]
+        public async Task Verify_AuthorSignedPackageWithTrustedCertificate_AuthorTag_Succeeds(string trust, bool fingerPrintOption)
```

Comment:
What's `Tag` in this context?

nit: It's fingerprint, not `fingerPrint`, it's 1 word.

Reply:
I renamed to `AuthorElement`, please check below screenshot for what it refers. Alternatively I could rename to `AuthorItem`.
![image](https://user-images.githubusercontent.com/8766776/150033219-02a5d960-f4e0-424d-9fee-ecfc122fc0c0.png)

Reply:
Should it be WithAuthorTrustedSigner? 

Basically you have an author signed package and you are verifying it with an author trusted signer item. 

How about: Verify_AuthorSignedPackage_WithAuthorItemTrustedCertificate_Succeeds

AuthorTag/AuthorItem itself is not super obvious, the setup should be clearer.

Reply:
Sure, I renamed all new tests same pattern.